pkg/sdk/row_access_policies_gen_test.go

```diff
@@ -243,3 +247,55 @@ func TestRowAccessPolicies_Describe(t *testing.T) {
 		assertOptsValidAndSQLEquals(t, opts, "DESCRIBE ROW ACCESS POLICY %s", id.FullyQualifiedName())
 	})
 }
+
+func TestRowAccessPolicyDescription_Arguments(t *testing.T) {
```

Comment:
can we get a signature like (A NUMBER(32, 4)) ? If so, then the implementation needs to be adjusted.

Reply:
No.